vertical-pod-autoscaler/README.md

```diff
@@ -112,6 +112,16 @@ in the `kube-system` namespace. It also generates
 and uploads a secret (a CA cert) used by VPA Admission Controller when communicating
 with the API server.
 
+To print YAML contents with all resources that would be understood by
+`kubectl diff|apply|...` commands, you can use
+
+```
+./hack/vpa-process-yamls.sh noop
```

Comment:
`Noop` is not prarticularly intuitive. What about `print`?

Reply:
Hi @mwielgus,

yes `print` is a lot better. I have updated the Pr (commits, subject and description with `push --force` command). 

Please let me know if there is something else I need to change or improve (I am not a native English speaker.)

Thanks a lot.